### PR TITLE
Adding Getting-Started-Guide (Issue: https://github.com/tektoncd/website/issues/17)

### PR DESCRIPTION
**Pull request by popcor255**
# Changes

# Submitter Checklist

These are the criteria that every PR should meet, please check them off as you
review them:

- [x] Includes [docs](https://github.com/tektoncd/community/blob/master/standards.md#principles) (if user facing)
- [x] Commit messages follow [commit message best practices](https://github.com/tektoncd/community/blob/master/standards.md#commit-messages)

_See [the contribution guide](https://github.com/tektoncd/website/blob/master/CONTRIBUTING.md)
for more details._

### CODE REVIEW

File: content/en/docs/install.md [hunk 1]
```diff
@@ -0,0 +1,128 @@
...
+weight: 20
+menu:
+  main:
+    weight: 20
+---
+
+To add Teketon to the Kubernetes Cluster on version 1.11 or later grant cluster-admin permissions to the current user.
+
+* [Installing Tekton Pipelines](#installing-tekton-pipelines)
+* [Installing Tekton PIpelines on OpenShift/MiniShift](#installing-tekton-pipelines-on-openshiftminishift)
```
Comment (vdemeester) [inline content/en/docs/install.md]: typo here `PIpelines` :stuck_out_tongue:

File: content/en/docs/install.md [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+   append the `--watch` flag to view the component's status updates in real
+   time. Use CTRL + C to exit watch mode.
+
+You are now ready to create and run Tekton Pipelines:
+
+- See [Tekton Pipeline tutorial](./tutorial.md) to get started.
+- Look at the
+  [examples](https://github.com/tektoncd/pipeline/tree/master/examples)
+
+### Installing Tekton Pipelines on OpenShift/MiniShift
```
Comment (vdemeester) [inline content/en/docs/install.md]: I think we should also talk about the Operator in the case of OpenShift :angel: 
`Minishift` is running OpenShift 3.x which is theoretically too old to support Tekton, so I wonder if we should document it :sweat_smile:

File: content/en/docs/install.md [hunk 3]
```diff
@@ -0,0 +1,128 @@
...
+data:
+  default-service-account: "tekton"
+  default-timeout-minutes: "20"
+  default-pod-template: |
+    annotations:
+      cluster-autoscaler.kubernetes.io/safe-to-evict: 'false'
+```
+
+*NOTE:* The `_example` key in the provided [config-defaults.yaml](./../config/config-defaults.yaml)
+file contains the keys that can be overriden and their default values.
```
Comment (vdemeester) [inline content/en/docs/install.md]: I think this is misplaced (or shouldn't be there)

Comment (popcor255) [reply content/en/docs/install.md]: ok, ill remove it.

File: content/en/docs/install.md (hunk 1 above)
Comment (popcor255) [reply content/en/docs/install.md]: lol, 😂 I fixed it.

File: content/en/docs/install.md (hunk 2 above)
Comment (popcor255) [reply content/en/docs/install.md]: Yeah, I was playing around with openshift 4 (CRC). Ill change it to be more relevant.

**Comment by popcor255** [comment]
Trying to address https://github.com/tektoncd/website/issues/16

**Comment by popcor255** [comment]
Closing until merge completed.

**Comment by popcor255** [comment]
This is so behind. Its not even funny.

**Pull request closed without merging**